fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -14,9 +14,6 @@ class FMCMainDisplay extends BaseAirliners {
         this._messageQueue = new A32NX_MessageQueue(this);
```

Comment:
For A32NX_FMCMainDisplay

Ctrl+F "return Fmgc.WayPointEntryUtils"
Compare the static function to the now dead code
Remove all dead code and ensure that any bug fixes (i.e. to regex matching) is not regressed in WayPointEntryUtils